app/views/foods/foods_search.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Search Results</h1>
+
+<% @foods.each do |food| %>
+  <%= food.total_hits%>
+<% end %>
+
+<% @foods.each do |food| %>
+  <% food.foods_list.first(10).each do |item|%>
+    <%= item[:description] %>
+    <%= item[:foodCode] %>
+<% end %>
```

Comment:
Instead of limiting here in the view, I'd suggest limiting how much you'd like to get back from the api, or at least limiting before iterating through in your facade to convert the json object into a food object. I suggest this because if you only want 10 food items, then you just took the entire payload, converted them to food objects, and THEN limited. That was a lot of conversion for nothing, so if you can limit before conversion, that will save some computing time.